docs/features/creating_container.md

```diff
@@ -91,19 +91,32 @@ func TestIntegrationNginxLatestReturn(t *testing.T) {
 
 _Testcontainers for Go_ allows you to define your own lifecycle hooks for better control over your containers. You just need to define functions that return an error and receive the Go context as first argument, and a `ContainerRequest` for the `Creating` hook, and a `Container` for the rest of them as second argument.
 
-You'll be able to pass multiple lifecycle hooks at the `ContainerRequest` as an array of `testcontainers.ContainerLifecycleHooks`, which will be processed one by one in the order they are passed.
-
-The `testcontainers.ContainerLifecycleHooks` struct defines the following lifecycle hooks, each of them backed by an array of functions representing the hooks:
+You'll be able to pass multiple lifecycle hooks at the `ContainerRequest` as an array of `testcontainers.ContainerLifecycleHooks`. The `testcontainers.ContainerLifecycleHooks` struct defines the following lifecycle hooks, each of them backed by an array of functions representing the hooks:
 
 * `PreCreates` - hooks that are executed before the container is created
 * `PostCreates` - hooks that are executed after the container is created
 * `PreStarts` - hooks that are executed before the container is started
 * `PostStarts` - hooks that are executed after the container is started
+* `PostReadies` - hooks that are executed after the container is ready
 * `PreStops` - hooks that are executed before the container is stopped
 * `PostStops` - hooks that are executed after the container is stopped
 * `PreTerminates` - hooks that are executed before the container is terminated
 * `PostTerminates` - hooks that are executed after the container is terminated
 
+_Testcontainers for Go_ defines some default lifecycle hooks that cannot be skipped, but you can add your own hooks to the default ones. The execution order will be the following:
+
+1. default `pre` hooks.
+2. user-defined `pre` hooks.
+3. user-defined `post` hooks.
+4. default `post` hooks.
+
+Inside each group, the hooks will be executed in the order they were defined.
+
+!!!info
+	The default hooks are for logging (applies to all hooks), customising the Docker config (applies to the pre-create hook) and copying files in to the container (applies to the post-create hook).
```

Comment:
I'm not sure what this means: "applies to all hooks" or "applies to pre-create hook". 
Neither do I see why it is relevant to the developers.

Reply:
I added where those hooks are applied, as documentation. I think it'd be clearer using passive style here:

```diff
- The default hooks are for logging (applies to all hooks), customising the Docker config (applies to the pre-create hook) and copying files in to the container (applies to the post-create hook).
+ The default hooks are for logging (applied on all hooks), customising the Docker config (applied on the pre-create hook) and copying files in to the container (applied to the post-create hook).
```